packages/medusa/src/api-v2/admin/fulfillments/route.ts

```diff
@@ -0,0 +1,31 @@
+import { createFulfillmentWorkflow } from "@medusajs/core-flows"
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "../../../types/routing"
+import { refetchFulfillment } from "./helpers"
+import { AdminCreateFulfillmentType } from "./validators"
+
+export const POST = async (
+  req: AuthenticatedMedusaRequest<AdminCreateFulfillmentType>,
+  res: MedusaResponse
+) => {
+  const { result: fullfillment, errors } = await createFulfillmentWorkflow(
```

Comment:
Typically the workflows take an array, should we do the same for a fulfillment?

Reply:
same reason as above, since this calls an external provider, we're keeping them non-bulk.

Reply:
Ok make sense 👍